assets/launcher/lang/sklmessages_fil_PH.properties

```diff
@@ -129,15 +129,15 @@ manager.profile.menu.edit=Baguhin
 manager.profile.menu.duplicate=Duplikahin
 manager.profile.menu.delete=Alisin
 
-manager.editor.header.title=Edit Installation
-manager.editor.footer.button.save=I-ligtas
```

Comment:
"I-ligtas" means to save someone, like in a life-threatening situation (e.g., "I-liligtas kita" = "I will save you"). For saving progress for example, "I-save" is used instead. Additionally, "I-ligtas" is grammatically incorrect; the proper form is "Iligtas" without the hyphen.